modules/benchmark/inst/scripts/benchmark.demo.Rmd

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
FOR NEXT RELEASE: You might migrate this file in the future. If it's a working shiny app move it to the shiny folder. If it's a vignette then it belongs in a specific folder (though I don't know if the R build system is set up to handle shiny vignettes, so you'll eventually want a non-shiny vignette as well for all us PDF-reading dinosaurs)